webviews/initialization_screen/index.html

```diff
@@ -1,41 +1,47 @@
-<link rel="stylesheet" href="{{styles.css}}">
-
-<center>
-    <img src="{{/assets/chrome.png}}" height="5%" width="5%">
-    <h1 align>New Chrome Extension</h1>
-</center>
-<hr>
+<link rel="stylesheet" href="{{styles.css}}" />
 <body>
-    <label for="project_name">Project Name:</label>
-    <input type="text" id="project_name" required>
-
-    <br><br>
-
-    <label for="extension_type">Extension Type <small>[<a href="https://github.com/gadhagod/chrome-extension-cli-client/blob/master/templates/README.md#templates">?</a>]</small>:</label>
-
-    <select id="extension_type">
-        <option value="popup">Popup</option>
-        <option value="override_page">Override page</option>
-        <option value="devtools">Dev tools</option>
-    </select>
-
-    <br><br>
-
-    <label id="location_label" for="choose_location">Location: </label>
-    <button id="choose_location" onclick="getLocation()">Choose</button>
-
-    <br><br>
-
+  <div class="container">
+    <div class="header">
+      <div class="icon">
+        <img src="{{/assets/chrome.png}}" alt="icon" />
+      </div>
+      <p class="title">Create New Chrome Extension</p>
+    </div>
+    <div class="project-form">
+      <div class="inputs">
```

Comment:
Can we keep the indentation to 4 spaces please?

Reply:
sure, I guess it was my editors default setting and i didn't pay much attention to it